call-home/src/transmitter/encrypt.rs

```diff
@@ -0,0 +1,52 @@
+use crate::{common::constants::*, Report};
```

Comment:
nit: should this file be called something like `encryption` rather than `encrypt`, `encrypt::encrypt()` doesn't sound good to me